docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,307 @@
+to understand the problem.  Blueprints types
+
+Currently, we have the blueprint types:
+
+ * Architectural Blueprint: Any blueprint changing or introducing a new core
+   feature or architectural change to Avocado.
+
+ * Process Blueprint: Any blueprint that is not implementing a new core
+   feature, but changing how the project works. This could be, for instance,
+   related to the repositories or processes.
+
+ * Meta Blueprint: A blueprint about blueprints. Like this one and any future
+   blueprint that changes our blueprints styles and methods.
+   
+ * Epic Blueprint: A blueprint that is touching in multiple areas and is too
+   big to have all the documentation in one single blueprint. We could split
+   epic blueprints into smaller blueprints or issues (if they are small and
+   easy to understand).
+
+ * Utility Blueprint: A blueprint with the intent to describe a new utility
+   module or a new plugin.
+
+Headers
+-------
+
+Python PEPs (Python Enhancement Proposals) uses RFC822 for describing the
+headers. This could be useful here too, especially when parsing those headers
+to display our list of blueprints with the current status.
+
+The current list of items of our blueprint headers is below:
+
+ * Number: Usually, the blueprint number in the format BPXXX
+
+ * Title: A short descriptive title, limited to 80 characters
+
+ * Author: The author or authors of blueprint 
+
+ * Reviewers: All reviewers that approved and helped during the review process
+
+ * Created: Date string when the blueprint first draft was submitted
+
+ * Type: One of the types described during the previous section
+
+ * Status: One of the types described during the next section
+
+Here is an example of a header::
+
+   :Number: BP001
+   :Title: Configuration by convention
+   :Author: Beraldo Leal <bleal@redhat.com>
+   :Reviewers: Cleber Rosa, Lukáš Doktor and Plamen Dimitrov
+   :Created: 06-Dec-2019
+   :Type: Epic Blueprint
+   :Status: WIP
+
+Blueprint statuses
+------------------
+
+ * Draft: All blueprints should be created in this state. This means the
+   blueprint is accepting comments, and probably there is a discussion
+   happening. Blueprints in draft mode can be part of our repository. 
+
+ * Approved: Blueprint was approved after discussions, and all suggestions are
+   already incorporated on the document. Nobody has started working on this
+   yet.
+
+ * WIP: This status is not about the blueprint itself, but about the proposal
+   that is the subject of the BP. This means that the blueprint was approved,
+   and someone is already working on implementing it. A BP status can change
+   from Draft to WIP if the work has started already. 
+
+ * Implemented: This means the BP is already implemented and delivered to the
+   Avocado’s master branch. 
+
+ * Rejected: Rejected status means the idea was not implemented because it
+   makes no sense or has some technical limitations.
```

Comment:
"makes no sense" is too strong, I would go for "wasn't approved by everyone or has some technical limitations".